### ISSUE TITLE
Creating suggestions without translations should not trigger warnings in console

### ISSUE THREAD

**Issue by petervdn** [mediamonks/kiai#6]

**Comment by n0cha** [comment]
Agreed. But should we not have warnings at all then for missing translations?
Or should we not automatically translate suggestions?
Or should only suggestions not trigger translation warnings?

**Comment by petervdn** [comment]
If supplying a regular text is one of the intended ways to use that method (which it is, and should be), then i don't think there should be warnings at all.

In my opinion the main issue is that the parameter can be multiple things (a text or a key to a translatable text), and i don't agree on that approach (same for the say method). It's shorter, but i think it gives more problems than it solves.
